src/components/PlioQuestion.vue

```diff
@@ -218,13 +227,88 @@ export default {
           this.hideSkipButton();
           this.isAnswerSubmitted = true;
           this.disableRadioButtons();
-        }, 200);
+        });
       }
 
       this.text = "";
       this.show = true;
       document.querySelector("body").classList.add("overflow-hidden");
       this.newProgressBarInfo['progressPercent'] = this.progressBarInfo['progressPercent']
+      this.handleImage()
+      this.styleQuestion()
+      this.styleOptions()
+      this.renderContent()
+    },
+
+    handleImage(){
+      this.$nextTick(() => {
+        // get question element
+        var question = document.getElementById('question')
+        if (question == null) return
+        // extract all <p> tags from the question element
+        var p_tags = question.querySelectorAll('p')
+        if (p_tags == null || p_tags.length == 0) return
+        // find which <p> tag has an <img> tag, cut it and
+        // paste it right outside, just below it.
+        p_tags.forEach(p_tag => {
+          var img_tag = p_tag.querySelector('img')
+          if (img_tag != null) {
+            p_tag.removeChild(img_tag)
+            p_tag.parentNode.insertBefore(img_tag, p_tag.nextSibling)
+            img_tag.setAttribute("id", "question_image")
+            // scale the image such that width is 1/3rd of screen width
+            // maintaining the aspect ratio
+            var heightToWidth = (img_tag.offsetHeight/img_tag.offsetWidth)
+            img_tag.style.width = String(window.screen.availWidth/3) + 'px'
+            img_tag.style.height = String(img_tag.offsetWidth * heightToWidth) + 'px'
+          }
+        })
+      })
+    },
+
+    renderContent(){
+      // force rendering of question and option texts
+      this.$nextTick(() => {
+        MathJax.Hub.Queue(["Typeset",MathJax.Hub,"question"]);
+        MathJax.Hub.Queue(["Typeset",MathJax.Hub,"option"]);
```

Comment:
Here, `question` and `option` are the `id` attributes of the tags that need to be rendered using MathJax